### PR TITLE
$ dove --version

### PR DESCRIPTION
**Pull request by vladimirovmm**

### CODE REVIEW

File: common/git-hash/src/lib.rs [hunk 1]
```diff
@@ -53,3 +93,93 @@ fn opt_tokenize(s: Option<String>) -> Result<TokenStream, LexError> {
...
+                let hash = &source[position + 1..];
+                { &hash[..7] }.to_string()
+            });
+            (branch, hash)
+        }
+
+        let toml_string = std::fs::read_to_string("Cargo.lock").unwrap_or_default();
+        toml_string
+            .split("[[package]]")
+            .find(|package| package.contains(r#"name = "move-stdlib""#))
```
Comment (RIg410) [inline common/git-hash/src/lib.rs:133]: We need a version of our standard library, not diem.
See https://github.com/pontem-network/move-tools/blob/master/dove/src/cmd/init.rs#L20

**Review by RIg410 — CHANGES_REQUESTED** [review]

File: common/git-hash/src/lib.rs (hunk 1 above)
Comment (vladimirovmm) [reply common/git-hash/src/lib.rs:118]: @RIg410 Thanks. Fixed it

File: common/git-hash/src/lib.rs [hunk 2]
```diff
@@ -31,13 +31,44 @@ pub fn git_hash_short(_: TokenStream) -> TokenStream {
...
-    crate_version_with(cmd::git_hash()).unwrap()
+pub fn crate_version_with_git_hash_short(_: TokenStream) -> TokenStream {
+    crate_version_with(cmd::git_hash_short()).unwrap()
 }
 
+/// Dove version
 #[proc_macro]
-pub fn crate_version_with_git_hash_short(_: TokenStream) -> TokenStream {
-    crate_version_with(cmd::git_hash_short()).unwrap()
+pub fn dove_version(_: TokenStream) -> TokenStream {
```
Comment (fzzr-) [inline common/git-hash/src/lib.rs:40]: Plz rename it back to `crate_version`.

File: common/git-hash/src/lib.rs [hunk 3]
```diff
@@ -31,13 +31,44 @@ pub fn git_hash_short(_: TokenStream) -> TokenStream {
 }
 
 #[proc_macro]
-pub fn crate_version_with_git_hash(_: TokenStream) -> TokenStream {
```
Comment (fzzr-) [inline common/git-hash/src/lib.rs:34]: Plz return removed fn `crate_version_with_git_hash `.

File: common/git-hash/src/lib.rs (hunk 2 above)
Comment (vladimirovmm) [reply common/git-hash/src/lib.rs]: @fzzr- Thanks. Fixed it

File: common/git-hash/src/lib.rs (hunk 3 above)
Comment (vladimirovmm) [reply common/git-hash/src/lib.rs:34]: @fzzr- Thanks. Fixed it

File: common/git-hash/src/lib.rs (hunk 2 above)
Comment (vladimirovmm) [reply common/git-hash/src/lib.rs]: @fzzr- Thanks. Fixed it

**Review by fzzr- — APPROVED** [review]

**Review by RIg410 — APPROVED** [review]

**Pull request merged**